.gitignore

```diff
@@ -3,6 +3,7 @@
 /bower_components
 /config/app.php
 /logs
+xdebug.log
```

Comment:
Un-needed - Should be included by the `/logs` line above, yeah?

Reply:
My bad. I started with `xdebug.log` in the root and then ran into issues (the file has to exist but I wanted to ignore changes to it). I should have moved the PR to draft but GitHub didn't make it easy so I just trodded on. About to udpate this PR with several local development changes (that seem to be related).